### PR TITLE
Pin Wasmtime to 0.8.0, as the rolling "dev" is no longer present

### PR DESCRIPTION
**Pull request by kripken**
See https://github.com/bytecodealliance/wasmtime/issues/714 more.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -426,9 +426,10 @@ jobs:
       - run:
           name: get wasmtime
           command: |
-            wget https://github.com/CraneStation/wasmtime/releases/download/dev/wasmtime-dev-x86_64-linux.tar.xz
-            tar -xf wasmtime-dev-x86_64-linux.tar.xz
-            cp wasmtime-dev-x86_64-linux/wasmtime ~/vms
+            # use a pinned version due to https://github.com/bytecodealliance/wasmtime/issues/714
```
Comment (sbc100) [inline .circleci/config.yml]: I think using a pinned copy is always makes for testing anyway,  So I wouldn't include this comment.. no explanation needed :)

File: .circleci/config.yml [hunk 2]
```diff
@@ -426,9 +426,10 @@ jobs:
...
+            wget https://github.com/bytecodealliance/wasmtime/releases/download/v0.8.0/wasmtime-v0.8.0-x86_64-linux.tar.xz
+            tar -xf wasmtime-*-x86_64-linux.tar.xz
+            cp wasmtime-*-x86_64-linux/wasmtime ~/vms
```
Comment (sbc100) [inline .circleci/config.yml]: I wouldn't use wild card here, just duplicate the version, or use an environment variable.  e.g:

```
VERSION=0.8.0
wget https://github.com/bytecodealliance/wasmtime/releases/download/v0.8.0/wasmtime-v$VERSION-x86_64-linux.tar.xz
cp wasmtime-dev-x86_64-linux/wasmtime ~/vms
tar -xf wasmtime-$VERSION-x86_64-linux.tar.xz
cp wasmtime-$VERSION-x86_64-linux/wasmtime ~/vms
```

File: .circleci/config.yml (hunk 1 above)
Comment (kripken) [reply .circleci/config.yml]: Well, at least the comment gives context for why it is as it is... but yeah, I get the argument for always pinning (even if I don't completely agree with it).

File: .circleci/config.yml (hunk 2 above)
Comment (kripken) [reply .circleci/config.yml]: Changed to an env var, thanks.

**Pull request merged**